src/agents/swarm_orchestrator.py

```
@@ -0,0 +1,778 @@
+                        output=None,
+                        error="Cancelled due to fail_fast policy",
+                        attempts=0,
+                        timed_out=False,
+                        duration_ms=0,
+                        operation_id=operation_id,
+                        correlation_id=correlation_id,
+                        sub_operation_id=f"{operation_id}:{agent_name}",
+                    )
+
+        ordered_results = [completed[name] for name in agent_names if name in completed]
+        return ordered_results
```

Ruff (0.15.5)

[warning] Unnecessary assignment to `ordered_results` before `return` statement

Remove unnecessary assignment

(RET504)